portfolio/src/main/java/com/google/sps/servlets/RandomGameServlet.java

```diff
@@ -15,18 +15,34 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
+@WebServlet("/random-game")
+public final class RandomGameServlet extends HttpServlet {
+
+  private List<String> games;
+
+  @Override
+  public void init() {
+    games = new ArrayList<>();
```

Comment:
Consider initializing this with all the values in place instead of adding them one by one

Reply:
Done.